doc/nvim-tree-lua.txt

```diff
@@ -1639,6 +1639,11 @@ e.g. handler for node renamed: >
                 handler parameters: ~
                   {folder_name}  `{string}` Absolute path to the removed folder.
 
+- Event.OnAttachPost
```

Comment:
Do we need `On` prefix here? Seems like other events don't have those. `AttachPost` sounds bit off though. How about `TreeAttached`? Might want to make it `TreeAttachedPost` if there is a chance of `TreeAttachedPre` being added as well.

Reply:
That makes sense. Telescope uses attach and gitsigns only adds `on_` for the callback itself.

Reply:
If we ever handle multiple trees/tabs, that could be added.

Reply:
Thanks, one more option I thought of: `TreeCreated` - maybe it's bit more descriptive than `Attached`. Up to you.

Reply:
on_attach happens in the middle of things.

We might want a TreeCreated event before or a TreeReady event after if a need arises :shrug:

Reply:
That makes sense then. Thanks once again.

Reply:
Thank you for the diligence.

I'm just super-paranoid about not walling us into any corners with API or Events.

Whilst I have you... could I please ask for a sanity check / test for #1872

Off until next weekend :wave: